lib/exercise_cases.rb

```diff
@@ -1,2 +1,30 @@
 require 'ostruct'
 require 'json'
+
+class ExerciseCase < OpenStruct
+  def name
+    'test_%s' % description.gsub(/[^\w ?!]/, '').gsub(/[- ]/, '_')
```

Comment:
I think it is a separate commit, so it may be a simple process.  That was a great idea to separate it out like that!